src/main/resources/bidder-config/apacdex.yaml

```diff
@@ -0,0 +1,32 @@
+adapters:
+  apacdex:
+    modifying-vast-xml-allowed: false
+    endpoint: http://useast.quantumdex.io/auction/pbs
+    aliases:
+      quantumdex:
+        usersync:
+          url: https://sync.quantumdex.io/usersync/pbs?gdpr={{gdpr}}&gdpr_consent={{gdpr_consent}}&us_privacy={{us_privacy}}&r=
+          redirect-url: /setuid?bidder=quantumdex&gdpr={{gdpr}}&gdpr_consent={{gdpr_consent}}&us_privacy={{us_privacy}}
+          cookie-family-name: quantumdex
+      valueimpression:
+        usersync:
+          url: https://sync.quantumdex.io/usersync/pbs?gdpr={{gdpr}}&gdpr_consent={{gdpr_consent}}&us_privacy={{us_privacy}}&r=
+          redirect-url: /setuid?bidder=quantumdex&gdpr={{gdpr}}&gdpr_consent={{gdpr_consent}}&us_privacy={{us_privacy}}
```

Comment:
Is `quantumdex ` need to be replaced with `valueimpression` here?

Reply:
I had the same question but in go `quantumdex` 
https://github.com/prebid/prebid-server/pull/2097/files#diff-e3ddc5c58f8470f02b08100357f822cdc75d2c0692d1d9ba2fdd608c739df905R19

Reply:
Then remove `url` and `redirect-url` properties from aliases.